### PR TITLE
feat: dummy voting space

### PR DESCRIPTION
**Pull request by dtscalac**
# Description

Voting space screen with dummy favorite proposal management.

## Related Issue(s)

#738

## Screenshots

Screenshot 2024-09-09 at 12 19 50

Screenshot 2024-09-09 at 12 18 00

## Please confirm the following checks

* [x] My code follows the style guidelines of this project
* [x] I have performed a self-review of my code
* [x] I have commented my code, particularly in hard-to-understand areas
* [x] I have made corresponding changes to the documentation
* [x] My changes generate no new warnings
* [x] I have added tests that prove my fix is effective or that my feature works
* [x] New and existing unit tests pass locally with my changes
* [x] Any dependent changes have been merged and published in downstream module

### CODE REVIEW

File: catalyst_voices/lib/pages/voting/voting_page.dart [hunk 1]
```diff
@@ -1,17 +1,190 @@
...
   Widget build(BuildContext context) {
-    return Container(
-      color: Colors.cyan,
-      alignment: Alignment.center,
-      child: Text(
-        'Voting',
-        style: Theme.of(context).textTheme.titleLarge,
+    return Padding(
+      padding: const EdgeInsets.symmetric(horizontal: 32),
+      child: ListView(
```
Comment (damian-molinski) [inline catalyst_voices/lib/pages/voting/voting_page.dart:66]: you should be able set padding as `ListView` parameter

Comment (dtscalac) [reply catalyst_voices/lib/pages/voting/voting_page.dart:66]: Ah true, first I added a column, wrapped it with padding, then replaced column with a list and padding stayed. Fixed.

**Review by damian-molinski — APPROVED** [review]
lgtm

**Review by digitalheartxs — DISMISSED** [review]
🎉

**Review by digitalheartxs — APPROVED** [review]

**Pull request merged**